### DISCUSSION TITLE
Checkbox filters

### DISCUSSION THREAD

**Discussion by artem-man** [rappasoft/laravel-livewire-tables#289] [Show and tell]
Hi Anthony. 
I have seen many requests for adding checkbox control in filters. Please see my path that add checkbox filters support (only bootstrap-4 theme done at this time). And thank you for your component )
[filters.patch.gz](https://github.com/rappasoft/laravel-livewire-tables/files/6511818/filters.patch.gz)

Usage:
FilterSelect::make('name')->setOptions($array_of_options);
FilterCheckbox::make('name')->setOptions($array_of_options);

Rest the same

PS
I forgot add support for pills, please use $showFilters = false; during the test

**Comment by rappasoft** [comment]
Thanks i'll check it out!

**Reply by artem-man** [reply]
Please see updated path with few fixes, and I think improvements in visual
[filters.patch.gz](https://github.com/rappasoft/laravel-livewire-tables/files/6518900/filters.patch.gz)

**Comment by artem-man** [comment]
Also, please see the following patch. This path call format callback function with filled column

[format-callback.patch.gz](https://github.com/rappasoft/laravel-livewire-tables/files/6518966/format-callback.patch.gz)

**Comment by lrljoe** [comment]
This is present in v2+
